tests/helper/shell_process.py

```diff
@@ -0,0 +1,13 @@
+import subprocess
+
+
+
+def shell_cmd(args):
+    """Helper Function for running subprocess"""
+    child = subprocess.Popen(args, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+    try:
+        result, errs = child.communicate(timeout=60)

```

Comment:
Maybe we should consider making shell_cmd take in a timeout as a parameter to allow for commands that should take longer or shorter to execute.